frontend/ui/uimanager.lua

```diff
@@ -374,6 +378,10 @@ end
 function UIManager:sendEvent(event)
     if #self._window_stack == 0 then return end
 
+    if type(event) == "string" then
```

Comment:
Is this change introduced to avoid writing `Event:new`? It can make the code a little bit harder to following because the UI event loop also handles events as strings (for example "Resume"), which is unfortunately a legacy tech dept that needs to be cleaned up. It's unclear from the code to know whether the event will be sent as a string or Event.

I would be more comfortable merging this change if: 1. we unify all events in the system to be of Event type. 2. make this method also supports event argument passing to be feature comparable with Event type.